configure

```diff
@@ -8,9 +8,9 @@
 # soname version
 #
 # arb     => soname
-# 2.7.0   => 0.0.0
+# 2.8.0   => 0.0.0
 ARB_MAJOR=2
-ARB_MINOR=7
+ARB_MINOR=8
```

Comment:
@fredrik-johansson not sure whether I should be doing this as part of a PR, feel free to revert this change.

Reply:
I update the version number when making a new release.